src/test/test-manager.c

```diff
@@ -0,0 +1,20 @@
+#include "manager.h"
+#include "tests.h"
+
+TEST(manager_taint_string) {
+        Manager m = {};
+
+        _cleanup_free_ char *a = manager_taint_string(&m);
+        assert_se(a);
+        log_debug("taint string w/o split-usr: '%s'", a);
+        /* split-usr is the only one that is cached in Manager, so we know it's not present */
```

Comment:
Shouldn't this check what the value is? It's not enforced besides the taint, no, so it could be already true here?